### PR TITLE
Reactjs and  Nodejs starterkits

### LINKED ISSUE #9: Create NodeJS starter-kit

### PR DESCRIPTION
**Pull request by WSMathias**

### CODE REVIEW

File: starterkits/react/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+x-app-envsfile: &app-envsfile .env
+
+x-app-networks: &app-networks
+  - default
+
+#_______________________________________________ Services __________________________________________________#
+services:
+  #---------------------------------- react-dev-server -----------------------------------------#
+  react:
+    image: "srijanlabs/node:${NODE_VERSION}-10-local"
```
Comment (mountain-nerd) [inline starterkits/react/docker-compose.yml:27]: I think this would break.

File: starterkits/node-express-postgres/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+ARG NODE_VERSION=12
+FROM srijanlabs/node:${NODE_VERSION}-10-dev as builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm i 
+
+FROM srijanlabs/nginx:10
```
Comment (mountain-nerd) [inline starterkits/node-express-postgres/Dockerfile]: This is an nginx image, is that intended

**Comment by mountain-nerd** [comment]
@WSMathias the node-express starterkit has bunch of controllers added, can we make it simple and only ship server.js or index.js?

**Comment by WSMathias** [comment]
> @WSMathias the node-express starterkit has bunch of controllers added, can we make it simple and only ship server.js or index.js?

I have maintained the folder structure for Sequelize ORM and Passportjs for authentication.
It's a complete boilerplate.

**Review by mountain-nerd — COMMENTED** [review]
> > @WSMathias the node-express starterkit has bunch of controllers added, can we make it simple and only ship server.js or index.js?
> 
> I have maintained the folder structure for Sequelize ORM and Passportjs for authentication.
> It's a complete boilerplate.

I think having opinioin on passport and ORM does not sound that good as different teams might have a different preference, so to keep it simple lets just ship app.js. @vs4vijay thoughts?

File: starterkits/node-express-postgres/Dockerfile (hunk 2 above)
Comment (WSMathias) [reply starterkits/node-express-postgres/Dockerfile]: yes just to serve static files post-build.

**Review by mountain-nerd — CHANGES_REQUESTED** [review]
@WSMathias please check.

**Pull request merged**